.github/workflows/ports_psoc6.yml

```diff
@@ -81,52 +81,11 @@ jobs:
         echo " >> PSoC6 extended hardware tests"
         ./psoc6/run_psoc6_tests.sh -c --psoc6-hwext --dev0 ${devs0[0]} 
         echo " >> PSoC6 bitstream tests"
```

Comment:
Can you please update the text here as i2s tests?

Reply:
done!